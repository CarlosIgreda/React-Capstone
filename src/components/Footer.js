const Footer = () => (
  <footer className="footer">
    Created by Carlos Igreda under MIT License.
  </footer>
);

export default Footer;
