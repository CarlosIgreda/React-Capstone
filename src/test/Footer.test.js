import React from 'react';
import { render } from '@testing-library/react';
import Footer from '../components/Footer';

test('renders footer correctly', () => {
  const { container } = render(<Footer />);
  expect(container.firstChild).toMatchSnapshot();
});
