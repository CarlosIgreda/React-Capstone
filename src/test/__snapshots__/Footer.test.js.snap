// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer correctly 1`] = `
<footer
  class="footer"
>
  Created by Carlos Igreda under MIT License.
</footer>
`;
